### PR TITLE
DDF-5035 updated user guide documentation

### LINKED ISSUE #5034: Add documentation for Search Forms and Results Forms
### Description

Search forms and results forms are missing from the UI documentation. 

### Version
2.17.0

### Additional Information
see #5031 for placement within outline

at minimum: 
- Define the feature and it's purpose. 
- explain how to complete a task with the feature
  - creating, editing, sharing search & result forms

### LINKED ISSUE #5037: Document the help tool in Intrigue
### Description
Document the help tool in Intrigue. 

Currently the help tool in Intrigue is not mentioned in the documentation, and it is not clear to a user on what it's purpose is. The help tool is not highly visible.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Copy edits and revisions to User guide documentation.

- Adds sections for Search Forms and Result Forms
- updated screenshots
- Verified steps for uploading, querying, filtering
- Added new tasks
- Documented Map Tools widget
- Standardized and made fixes to outline

#### Select relevant component teams: 
@codice/docs 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@adimka
@ahoffer
@andrewkfiedler
@brendan-hofmann
@djblue
@lessarderic
@mcalcote
@ricklarsen - Documentation

#### How should this be tested?

Compare the documented steps against the attached task list: 

[task_list.txt](https://github.com/codice/ddf/files/3623076/task_list.txt)

#### What are the relevant tickets?
Fixes: #5035 
Fixes #5034 
Fixes #5037 

#### Screenshots
Rendered version: 
[docs.zip](https://github.com/codice/ddf/files/3623081/docs.zip)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+. Select *View Sharing*.
+.. To share by user role, set the drop-down menu to *Read Only* or *Read and Write* for each desired role. All users with that role will be able to view the workspace, but will be limited based on the permission assigned. No user will be granted the ability to share the workspace with additional users.
+.. To share with an individual user, add his/her email to the email list and set the drop-down menu to *Read Only*, *Read and Write*, or *Read, Write, and Share*.
+. Click *Apply*.
+
+.Remove Sharing on a Workspace
+. From the *Workspaces* view, select the *Options* menu (image:options-icon.png[options icon]) for the workspace in which sharing will be modified.
+. Select *View Sharing*.
+.. To remove the workspace from users with specific roles, set the drop-down menu to *No Access* for those roles.
+.. To remove individual users, remove the users' email addresses from the email list.
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc]: ✏️ `users'` -> `user's`

File: distribution/docs/src/main/resources/content/_using/sources-ui.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+:parent: Using ${catalog-ui}
+:summary: Uploading from ${catalog-ui}
+:order: 04
+
+== {title}
+
+${branding} can be configured to search multiple data sources simultaneously,
+or searches can be limited to specific sources as needed.
+
+To view the status of all configured sources, select *Sources* from the Navigation Menu..
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/content/_using/sources-ui.adoc]: ✏️ Extra `.` at the end.

File: distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc (hunk 1 above)
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc]: ❔ just verifying that you mean plural ownership with `users` here

File: distribution/docs/src/main/resources/content/_using/attribute-editor.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+:type: subUsing
+:status: published
+:parent: Pan to a Location
+:summary: Using the Attribute Editor
+:order: 00
+
+== {title}
+
+
+${catalog-ui} provides an upload editor form which allows users to customize the metadata of their
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_using/attribute-editor.adoc]: ❔ are `upload editor` and `attribute editor` the same thing? Seems like this should be `attribute editor`?

File: distribution/docs/src/main/resources/content/_using/attribute-editor.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+uploads. If enabled, it will appear alongside the upload dropzone and will displays a list of
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/attribute-editor.adoc]: fixed

File: distribution/docs/src/main/resources/content/_using/attribute-editor.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+attributes a that may be set.
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/attribute-editor.adoc]: fixed

File: distribution/docs/src/main/resources/content/_using/configuring-workspaces.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+. Click the save (image:save-icon.png[save icon]) icon next to the workspace title in the upper left corner.
+
+.Workspace Menu Options
+* To view a workspace's options from the *Workspaces* view, press the *Options* button (image:options-icon.png[options icon]) for the workspace.
+** *Save Workspace*: Save changes to the workspace.
+** *Run All Searches*: Start all saved searches within this workspace.
+** *Cancel All Searches*: Cancel all running searches.
+** *Open Workspace in New Tab*: Opens this workspace in a separate tab.
+** *View Sharing*: View and edit settings for sharing this workspace. Users must be signed in to share workspaces or view shared workspaces.
+** *View Details*: View the current details for a cloud-based workspace Users must be signed in to view workspace details.
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/configuring-workspaces.adoc]: Fixed

File: distribution/docs/src/main/resources/content/_using/create-a-search.adoc [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+.. Select the <<{using-prefix}search_types,type>> of search to perform: *Text*, *Basic*, or *Advanced*.
+. Results will be displayed in the *Search Tab*.
+. If too many or too few results are returned, edit the search parameters by selecting the *edit* icon.
+. Once the search is correct, select the *Save* icon next to the workspace name in the top bar.
+
+.Creating a New Search Into a New Workspace
+. Select the navigation menu icon.
+. Select *Workspaces*.
+. Enter search terms into the *Search* box.
+. A new workspace, by default called *New Workspace* will be created containing only the new search.
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/create-a-search.adoc]: fixed

File: distribution/docs/src/main/resources/content/_using/editing-searches.adoc [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+The available searches for a workspace can be viewed by clicking on the drop-down on the *Search* tab.
+
+.Viewing available searches.
+image::searches-dropdown.png[searches dropdown]
+
+.Search Menu Options
+At the bottom of each search is a list of options for the search.
+
+* *Run* &#9654;: Trigger this search to begin immediately.
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/editing-searches.adoc]: I was trying it out to render the icons. I can drop it if we don't like it

File: distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc]: revised for clarity

File: distribution/docs/src/main/resources/content/_using/sources-ui.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/sources-ui.adoc]: fixed

File: distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/sharing-workspaces.adoc]: It was a tortured sentence with 'individual' and the pluralization, so I made it all singular.

File: distribution/docs/src/main/resources/content/_using/attribute-editor.adoc (hunk 3 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_using/attribute-editor.adoc]: went with attribute editor

**Comment by ricklarsen** [comment]
build now

**Pull request merged**